akka-docs/src/main/paradox/stream/stream-flows-and-basics.md

```diff
@@ -51,8 +51,8 @@ Examples of operators are `map()`, `filter()`, custom ones extending @ref[`Graph
 junctions like `Merge` or `Broadcast`. For the full list of built-in operators see the @ref:[operator index](operators/index.md)
 
 
-When we talk about *asynchronous, non-blocking backpressure* we mean that the operators available in Akka
-Streams will not use blocking calls but asynchronous message passing to exchange messages between each other, and they
+When we talk about *asynchronous, non-blocking backpressure*, we mean that the operators available in Akka
+Streams will not use blocking calls but asynchronous message passing to exchange messages between each other. Instead, they
```

Comment:
You're right. How about 'Similarly' or 'In the same spirit'?

Reply:
Perhaps we should turn that sentence around a bit: 'asynchronous means' is a bit confusing because of the double meaning of 'means' anyway. Perhaps simply: `This way they can slow down a fast producer without blocking its thread.`?

Reply:
That's a good idea. 👍

Reply:
Done